en/identity-server/next/mkdocs.yml

```diff
@@ -769,7 +773,6 @@ nav:
           - Remove References to Deleted User Identities: deploy/configure/databases/remove-references-to-deleted-user-identities.md
         - Session persistence: deploy/configure/session-persistence.md
         - Clock tolerance: deploy/configure/clock-tolerance.md
-        - Email sending module : deploy/configure/email-sending-module.md
```

Comment:
Let's setup redirections, otherwise the existing links for this path in external resources will be broken

Reply:
We no longer need to display Email Sending Module under this section in the navigation bar.

Reply:
Even if the resource is there, since we has removed the path, if someone has used this link as a reference to email configuration, those links will start throwing 404.

To overcome this, we need to add a redirection[1], so that any references point to deploy/configure/email-sending-module will be redirected to guides/notification-channels/configure-email-provider.

[1] https://github.com/wso2/docs-is/blob/master/en/identity-server/next/mkdocs.yml#L110